tutorials/Schedule/shared_calendars.md

```diff
@@ -1,11 +1,15 @@
 # Shared calendars
 
-We have three google calendars to help you see events in your local time! If you open the links below, they will be in
+We have a google calendar for each Time Slot to help you see events in your local time! If you open the links below, they will be in
 universal coordinated time (UTC). If you click at the bottom to add to your Google Calendar, you should see them in the time zone of your google 
 calendar (probably yours)!
 
-[Zone 1/2](https://calendar.google.com/calendar/u/0/embed?src=c_20m8o03v9gf56avpkphsd3i9bk@group.calendar.google.com)
+[Zone 1](https://calendar.google.com/calendar/u/0/embed?src=c_20m8o03v9gf56avpkphsd3i9bk@group.calendar.google.com)
+
+[Zone 2](https://calendar.google.com/calendar/embed?src=c_8cb1f0fed705b06a794c89da599728c5c6df7063f8b9a9e18fb110ffa5868809%40group.calendar.google.com&ctz=UTC)
 
 [Zone 3](https://calendar.google.com/calendar/embed?src=c_r387vvjd66sg2o96cracfv5ioc%40group.calendar.google.com)
 
-[Zone 4/5](https://calendar.google.com/calendar/embed?src=c_ghv53rt8temq4gco5h0anp3ou4%40group.calendar.google.com)
+[Zone 4](https://calendar.google.com/calendar/embed?src=c_ghv53rt8temq4gco5h0anp3ou4%40group.calendar.google.com)
+
+[Zone 5](https://calendar.google.com/calendar/embed?src=c_8cb1f0fed705b06a794c89da599728c5c6df7063f8b9a9e18fb110ffa5868809%40group.calendar.google.com&ctz=UTC)
```

Comment:
looks good. are the other zones supposed to be empty?

Reply:
there should be a calendar for each zone. Is 1 missing? 

here's the link: https://calendar.google.com/calendar/embed?src=c_20m8o03v9gf56avpkphsd3i9bk%40group.calendar.google.com&ctz=America%2FDenver

Reply:
@iamzoltan  can you get this approved?